SimplyE/Colors.xcassets/disabledFieldText.colorset/Contents.json

```diff
@@ -1,23 +1,18 @@
 {
-  "info" : {
-    "version" : 1,
-    "author" : "xcode"
-  },
   "colors" : [
     {
-      "idiom" : "universal",
       "color" : {
         "color-space" : "srgb",
         "components" : {
-          "red" : "250",
           "alpha" : "1.000",
-          "blue" : "250",
-          "green" : "250"
+          "blue" : "0x66",
```

Comment:
if this is the default used by  light mode, shouldn't this stay at 0xFA?

Reply:
Do you mean the color should be #6363FA by stays at 0xFA?

Reply:
no I meant each RGB component was 250 (==0xFA) but now they are 0x66, so I was wondering why changing those since they are for light mode

Reply:
Oh, I deleted the `ColorBackground` in the Assets and somehow git thought its a rename, the original value has nothing to do with this color.